tests/Place/Canonical/CanonicalServiceTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\Canonical;
+
+use CultuurNet\UDB3\DBALTestConnectionTrait;
+use CultuurNet\UDB3\Event\ReadModel\Relations\Doctrine\DBALRepository;
+use CultuurNet\UDB3\Label\ReadModels\Relations\Repository\Doctrine\DBALReadRepository;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\InMemoryDocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\StringLiteral;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Types\Type;
+use PHPUnit\Framework\TestCase;
+
+class CanonicalServiceTest extends TestCase
+{
+    use DBALTestConnectionTrait;
+
+    private DBALDuplicatePlaceRepository $duplicatePlaceRepository;
+
+    private DocumentRepository $documentRepository;
+
+    private CanonicalService $canonicalService;
+
+    public function setUp(): void
+    {
+        $this->documentRepository = new InMemoryDocumentRepository();
+        $labels_relations = new Table('labels_relations');
```

Comment:
Casing of variables should always be `camelCasing`, so it should be `labelsRelations`